BUILD.gn

```diff
@@ -83,6 +83,10 @@ if (!is_android) {
       rebase_path(vulkan_data_dir, root_build_dir)
     ] + rebase_path(sources, root_build_dir)
 
+    if (is_win) {
+      args += [ "--no-path-prefix" ]
+    }
+
```

Comment:
Before my ICD cleanup commit the json didn't have a relative path, just "VkICD_mock_icd.dll"

So this should keep the behavior Angle was expecting.